lib/system.nim

```diff
@@ -551,6 +551,10 @@ when notJSnotNims and not defined(nimSeqsV2):
 when notJSnotNims:
   include "system/hti"
 
+  proc writeToStdErr(msg: cstring, length: int)
```

Comment:
> I'm not sure if these failed checks are related?

most likely related, and you should be able to reproduce the bug locally with smthg like
`testament pcat dll` or similar